### PR TITLE
Merged Mews.Fiscalizations.Uniwix with Mews.Fiscalizations.Italy

### PR DESCRIPTION
**Pull request by abdallahbeshi**
Next steps:
1. Add tests for the Italian fiscalization
2. Add GH secrets for the credentials
3. Add GH actions (pipelines) for Italy lib.
4. Update the documentation with code examples.

### CODE REVIEW

**Review by oleg-sokolansky — DISMISSED** [review]

File: src/Italy/Mews.Fiscalizations.Italy/Communication/Dto/InvoiceStateResult.cs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 using System;
 using Newtonsoft.Json;
 
-namespace Mews.Fiscalizations.Uniwix.Communication.Dto
+namespace Mews.Fiscalizations.Italy.Communication.Dto
```
Comment (marektresnak) [inline src/Italy/Mews.Fiscalizations.Italy/Communication/Dto/InvoiceStateResult.cs:4]: This is still specific to Uniwix, so I think it should at least be in a separate namespace.

File: src/Italy/Mews.Fiscalizations.Italy/Mews.Fiscalizations.Italy.csproj [hunk 2]
```diff
@@ -3,18 +3,25 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Authors>Mews</Authors>
-    <Description>Client for SDI - Italian fiscal registry.</Description>
-    <PackageTags>sdi fiscalization italy</PackageTags>
+    <Description>Client for SDI using Uniwix API. Used in italy for fiscal compliance.</Description>
```
Comment (marektresnak) [inline src/Italy/Mews.Fiscalizations.Italy/Mews.Fiscalizations.Italy.csproj:6]:
```suggestion
    <Description>Client for SDI using Uniwix API. Used in Italy for fiscal compliance.</Description>
```

**Review by marektresnak — APPROVED** [review]

**Comment by TashaBubo** [comment]
@marektresnak `tested`?

**Comment by marektresnak** [comment]
@TashaBubo Does it make sense for these libs? I don't think there's ever a situation that would require manual testing. This can only be manually tested when it's actually used in Mews, which requires a separate PR.

**Pull request merged**